### ISSUE TITLE
[Bug] No cleanup of CacheFiles in cache and destination directories

### ISSUE THREAD

**Issue by tyhawk** [HaveAGitGat/Tdarr#340]
Tdarr doesn't clean up errorous CacheFiles when copying from cache to destination directory failes.

I think it should delete these partial files when a copy fails.

```
-rwxrwxrwx 1 nobody nobody 538M  1 feb 12:03 MyVideoFile.mp4*
-rwxrwxrwx 1 nobody nobody 471M  1 feb 12:02 MyVideoFile-TdarrCacheFile-H4rAB7bgb.mp4*
-rwxrwxrwx 1 nobody nobody 341M  1 feb 12:02 MyVideoFile-TdarrCacheFile-TLVKSTrsT.mp4*
-rwxrwxrwx 1 nobody nobody 202M  1 feb 11:57 MyVideoFile-TdarrCacheFile-uCBASYS1M.mp4*
```

Tdarr 2.00.03, running in Docker (Arch Linux) in swarm mode.
Server: docker image
Nodes: mixture of docker and amd64 linux.

The cache directory is an NFS share mounted on all nodes and the server. I'm also seeing no cleanup in the cache directory. There are now dozens of transcoded en subrip temp files that are either never properly copied of failed during testing of some other issues I'm noticing.

**Comment by HaveAGitGat** [comment]
Thanks I'll take a look at this tomorrow.

**Comment by HaveAGitGat** [comment]
Fixed for the next version. Thanks.

**Comment by tyhawk** [comment]
Awesome! Thanks!

**Comment by HaveAGitGat** [comment]
2.00.04 is now in pre-release so you have to manually specify 2.00.04 in the Tdarr_Updater config (or specify the Docker tag)

```
{
  "platform_arch": "default",
  "modules": {
    "Tdarr_Desktop": {
      "enabled": true,
      "requiredVersion": "2.00.04",
      "currentVersion": ""
    },
    "Tdarr_Node": {
      "enabled": true,
      "requiredVersion": "2.00.04",
      "currentVersion": ""
    },
    "Tdarr_Server": {
      "enabled": true,
      "requiredVersion": "2.00.04",
      "currentVersion": ""
    },
    "Tdarr_WebUI": {
      "enabled": true,
      "requiredVersion": "2.00.04",
      "currentVersion": ""
    }
  }
}
```

**Comment by tyhawk** [comment]
I will upgrade to 2.00.04 tomorrow and test further. I will let you know if this issue has been resolved in my particular use case.

**Comment by tyhawk** [comment]
Solved in 2.00.04.
